tests/test_running_from_different_dir.py

```diff
@@ -0,0 +1,46 @@
+"""Check configs are picked up when running in different directory."""
+import subprocess
+from pathlib import Path
+from textwrap import dedent
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    "arg, cwd",
+    [
+        ("tests", Path.cwd()),
+        ("data", Path.cwd() / "tests"),
+        ("notebook_for_testing.ipynb", Path.cwd() / "tests/data"),
+    ],
+)
+def test_running_in_different_dir_works(arg: Path, cwd: Path) -> None:
+    """
+    Check .nbqa.ini config is picked up when running from non-root directory.
+
+    Parameters
+    ----------
+    arg
+        Directory or notebook to run command on.
+    cwd
+        Directory from which to run command.
+    """
+    with open(".nbqa.ini", "w") as handle:
+        handle.write(
+            dedent(
+                """\
+                [flake8]
+                addopts = --ignore=F401 \
+                """
+            )
+        )
+
+    output = subprocess.run(
+        ["nbqa", "flake8", arg], stderr=subprocess.PIPE, stdout=subprocess.PIPE, cwd=cwd
+    )
+
+    Path(".nbqa.ini").unlink()
```

Comment:
Nice. Writing using `Path.write_text` is concise

Reply:
Thanks! Agreed that it's more concise with .write_text, although if it's OK I'd still like to use dedent because otherwise I need to use `os.linesep` to make the Windows tests pass, which would hamper readability

Reply:
:thinking: windows tests are failing if I use write_text

Reply:
nevermind, failure was from passing a Path to subprocess, sorry for the noise :flushed: